cli/utils/ts-linter.js

```diff
@@ -19,6 +19,10 @@ function lintSync() {
   logger.info('Starting TSLint...');
 
   const spawnResult = spawn.sync('./node_modules/.bin/tslint', flags);
+  if (spawnResult.error) {
+    logger.error(spawnResult.error.message);
+    return process.exit(1);
```

Comment:
I cloned [the template](https://github.com/blackbaud/skyux-template); and pointed builder to `blackbaud/skyux-builder#npm5-v2`.

After installing the dependencies, I then typed `skyux build -s` and got an error from the tslinter: 

```
error: spawnSync ./node_modules/.bin/tslint ENOENT
```

I think this error is saying that `.bin/tslint` doesn't exist in that directory (relative to the cwd).

I ran across this issue in my `npm5` branch, too, and couldn't figure out a good solution.

Reply:
FYI, I'm running npm v4.6.1